packages/ui/src/components/TextArea/index.tsx

```diff
@@ -136,9 +136,13 @@ type TextAreaProps = {
    */
   helper?: ReactNode
   /**
-   * Number of rows to display. If 'auto', the textarea will grow with the content and won't be resizable.
+   * Number of rows to display. If 'auto', the textarea will grow with the content and won't be resizable **WARNING**: `auto` is deprecated - use prop `autoExpandMax` instead
    */
   rows?: number | 'auto'
```

Comment:
We should keep `auto`, and make `rowsMax` in the way that if we have `row="auto" maxRows={10}` it will follow the number of lines until the 10th one